[2025-06-19T11:08:19.507+0900] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-19T11:08:19.522+0900] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: image_classify_v1.check_file_exists_branch manual__2025-06-19T02:08:14.091306+00:00 [queued]>
[2025-06-19T11:08:19.529+0900] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: image_classify_v1.check_file_exists_branch manual__2025-06-19T02:08:14.091306+00:00 [queued]>
[2025-06-19T11:08:19.533+0900] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-06-19T11:08:19.546+0900] {taskinstance.py:2890} INFO - Executing <Task(_BranchPythonDecoratedOperator): check_file_exists_branch> on 2025-06-19 02:08:14.091306+00:00
[2025-06-19T11:08:19.558+0900] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=47215) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-19T11:08:19.560+0900] {standard_task_runner.py:72} INFO - Started process 47233 to run task
[2025-06-19T11:08:19.560+0900] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'image_classify_v1', 'check_file_exists_branch', 'manual__2025-06-19T02:08:14.091306+00:00', '--job-id', '1208', '--raw', '--subdir', 'DAGS_FOLDER/img_classify.py', '--cfg-path', '/tmp/tmp4_5vmyr0']
[2025-06-19T11:08:19.562+0900] {standard_task_runner.py:105} INFO - Job 1208: Subtask check_file_exists_branch
[2025-06-19T11:08:19.646+0900] {task_command.py:467} INFO - Running <TaskInstance: image_classify_v1.check_file_exists_branch manual__2025-06-19T02:08:14.091306+00:00 [running]> on host f6d4228a9eb8
[2025-06-19T11:08:19.971+0900] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='image_classify_v1' AIRFLOW_CTX_TASK_ID='check_file_exists_branch' AIRFLOW_CTX_EXECUTION_DATE='2025-06-19T02:08:14.091306+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-19T02:08:14.091306+00:00'
[2025-06-19T11:08:19.976+0900] {taskinstance.py:732} INFO - ::endgroup::
[2025-06-19T11:08:19.993+0900] {python.py:240} INFO - Done. Returned value was: get_file_info_list_task
[2025-06-19T11:08:19.994+0900] {branch.py:38} INFO - Branch into get_file_info_list_task
[2025-06-19T11:08:19.997+0900] {skipmixin.py:233} INFO - Following branch ('get_file_info_list_task',)
[2025-06-19T11:08:20.043+0900] {skipmixin.py:281} INFO - Skipping tasks [('end_task', -1)]
[2025-06-19T11:08:20.163+0900] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-06-19T11:08:20.165+0900] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=image_classify_v1, task_id=check_file_exists_branch, run_id=manual__2025-06-19T02:08:14.091306+00:00, execution_date=20250619T020814, start_date=20250619T020819, end_date=20250619T020820
[2025-06-19T11:08:20.317+0900] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-19T11:08:20.489+0900] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-19T11:08:20.496+0900] {local_task_job_runner.py:245} INFO - ::endgroup::
